confd/vars/main.yaml

```diff
@@ -20,6 +20,8 @@ kilda_zookeeper_reconnect_delay_ms_server42: 10000
 kilda_zookeeper_reconnect_delay_ms_speaker: 1000
 kilda_opentsdb_hosts: "opentsdb.pendev"
 kilda_opentsdb_port: "4242"
+kilda_victoriametrics_host: "victoriametrics.pendev"
+kilda_victoriametrics_port: "4242"
```

Comment:
Opentdstb and victoria have equal port 4242. As we can run both DBs for local env will we have problems with "Address 127.0.0.1:4242 is already in use"?